syntax = "proto3";

message LogRaw {
  string type = 1;
  uint64 log_index = 2;
  string transaction_hash = 3;
  uint32 transaction_index = 4;
  string address = 5;
  string data = 6;
  string indexed = 7;
  uint64 block_number = 8;
  uint64 block_timestamp = 9;
  string block_hash = 10;
  string item_id = 11;
  string item_timestamp = 12;
}
